packages/core/test/lib/create.js

```diff
@@ -124,32 +123,29 @@ describe("create", function() {
       err
     ) {
       if (err) return done(err);
+      const files = glob.sync(`${config.migrations_directory}${path.sep}*`);
 
-      dir.files(config.migrations_directory, function(err, files) {
-        if (err) return done(err);
+      const found = false;
+      const expectedSuffix = "_my_new_migration.js";
 
-        var found = false;
-        var expected_suffix = "_my_new_migration.js";
+      for (let i = 0; i < files.length; i++) {
+        const file = files[i];
```

Comment:
Did you know... you can just do:

```javascript
for (let file of files) {
  // ...
}
```

since it doesn't look like you're using `i` at all.